src/main/resources/templates/job.xml

```diff
@@ -2,6 +2,19 @@
     <description/>
     <keepDependencies>false</keepDependencies>
     <properties>
+        {% if (BUILD_PARAMETERS is defined) %}
```

Comment:
This needs to be changed to `{% if BUILD_PARAMETERS %}`.
Otherwise the generated XML will be like this when I don't provide any build parameters:
```
<flow-definition plugin="workflow-job@2.8">
    <description/>
    <keepDependencies>false</keepDependencies>
    <properties>
        
            <hudson.model.ParametersDefinitionProperty>
                <parameterDefinitions>
                    
                </parameterDefinitions>
            </hudson.model.ParametersDefinitionProperty>
            <org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty>
            <triggers/>
        </org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty>
    </properties>
...
```
which causes this exception on Jenkins, when I want to change the job config on the UI:
```
avax.servlet.ServletException: java.lang.Error: Failed to instantiate class hudson.model.ParametersDefinitionProperty from {"specified":true}
	at org.kohsuke.stapler.Stapler.tryInvoke(Stapler.java:796)
	at org.kohsuke.stapler.Stapler.invoke(Stapler.java:876)
	at org.kohsuke.stapler.MetaClass$5.doDispatch(MetaClass.java:236)
```


with `{% if BUILD_PARAMETERS %}`, it will be like this:
```
<flow-definition plugin="workflow-job@2.8">
    <description/>
    <keepDependencies>false</keepDependencies>
    <properties>
        
        <org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty>
            <triggers/>
        </org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty>
    </properties>
...
```
which is nice to not have any of the unnecessary things